build/sign.sh

```diff
@@ -44,4 +44,12 @@ echo RELOADAGENT | gpg-connect-agent
 for deb_file in $(find -regex ".*\.\(deb\)");do
   echo "===> Signing $deb_file"
   debsigs --sign=origin --verify --check -v -k ${GPG_MAIL} $deb_file
-done
\ No newline at end of file
+done
+
+# Sign TARGZ files
+for targz_file in $(find -regex ".*\.\(tar.gz\)");do
```

Comment:
-regex ".*tar.gz" ?

Reply:
it has the same effect, I followed the "same convention" as the other extensions